catalogue_api/api/src/test/scala/uk/ac/wellcome/platform/api/works/ApiErrorsTest.scala

```diff
@@ -235,21 +235,6 @@ class ApiErrorsTest extends ApiWorksTestBase {
     }
   }
 
-  it("returns Bad Request if you ask for a malformed identifier") {
-    ApiVersions.values.toList.foreach { version: ApiVersions.Value =>
```

Comment:
I had to update the elastic4s version to add the streams stuff and apparently that fixed this problem as well. Now the api return 404 regardless of the characters in the identifier

Reply:
💃

Reply:
Actually probably deleting this test is not the correct thing to do. I should rather change it to reflect the new behaviour.